src/app/components/planner-list/planner-list.component.ts

```diff
@@ -509,10 +515,31 @@ export class PlannerListComponent implements OnInit, AfterViewInit, DoCheck, OnD
         //Check if the work item meets the applied filters
         if(this.filterService.doesMatchCurrentFilter(item)){
           console.log('Added WI matches the applied filters');
+          let message = 'Work item ' + item.attributes['system.title'] + ' added.'
+          try {
+            this.notifications.message({
+              message: message,
+              type: NotificationType.SUCCESS
+
+            } as Notification);
+          } catch (e) {
+            console.log('Error displaying notification. Work item updated.')
```

Comment:
Added or updated? Better to append the `message` itself for consistency after the notification display error sentence, imo.